doc/sphinx/docs/python/tutorials/evolving_a_population.rst

```diff
@@ -12,8 +12,8 @@ an interior point optimizer or an evolutionary startegy solver, in pygmo you wil
 always have to call a method called ``evolve()`` to improve over your initial solutions,
 i.e. your *population*.
 
-The simplest way to evolve a population is to use directly the :class:`~pygmo.core.algorithm`
-method :class:`~pygmo.core.algorithm.evolve`
+The simplest way to evolve a population is to use directly the :class:`~pygmo.algorithm`
+method :func:`~pygmo.algorithm.evolve`
```

Comment:
Usually we put trailing ``()`` when referring to functions (so there's immediate visual feedback that we are talking about a function): ``algorithm.evolve()``